src/test/java/org/mule/module/apikit/UrlUtilsTestCase.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.apikit;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import org.junit.Test;
+import org.mule.module.apikit.api.UrlUtils;
+
+public class UrlUtilsTestCase {
```

Comment:
Could you please add a test where the base path has no `/` character?

Reply:
I think this is not a real case scenario using http, I will give it a try in Studio

Reply:
check the last assertion assertThat(UrlUtils.getRelativePath("api", "/api"), equalTo("/"));

Reply:
yeah, right, I was just thinking about what the method was doing... looks good!